test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-search-block-spec.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * External dependencies
```

Comment:
@worldomonation @bsessions85   -This is the WIP test case for pricing. Please ignore this one for now.